README.md

```diff
@@ -59,7 +59,7 @@ Usage
 
 |                 | Linux | macOS | Windows | Linux ARM | macOS ARM | Windows ARM |
 |-----------------|-------|-------|---------|-----------|-----------|-------------|
-| GitHub Actions  | ✅    | ✅    | ✅       | ✅¹       | ✅        | ✅²         |
+| GitHub Actions  | ✅    | ✅    | ✅       | ✅        | ✅        | ✅²         |
```

Comment:
Should the actual ¹ text be dropped below as well?

Reply:
That footnote still applies to Gitlab, who don't yet offer ARM runners.